src/stores/walletStore/walletStore.test.tsx

```diff
@@ -150,7 +150,7 @@ describe('walletStore', () => {
     announceProvider(metamask);
 
     await walletStore.connectWallet({
-      walletType: WalletTypes.EIP6963,
+      walletProvider: WalletProvider.EIP6963,
```

Comment:
Most of the diff is changing this name from `walletType` to `walletProvider`.

Then we can use `walletType` to refer to the individual wallet brands (MetaMask, HOT Wallet, etc.)